go.mod

```diff
@@ -71,7 +71,7 @@ require (
 	github.com/coinbase/rosetta-sdk-go v0.7.0 // indirect
 	github.com/confio/ics23/go v0.7.0 // indirect
 	github.com/containerd/continuity v0.3.0 // indirect
-	github.com/cosmos/btcutil v1.0.4 // indirect
+	github.com/cosmos/btcutil v1.0.4
```

Comment:
should this be moved to the direct stanza?

Reply:
Should I do this for all lines that no longer have the indirect comment? There appears to be a few of them

Reply:
Yes, please, that would be great